deploy/olm-catalog/service-telemetry-operator/1.1.0/service-telemetry-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -15,6 +15,8 @@ metadata:
             "graphingEnabled": false,
             "highAvailabilityEnabled": false,
             "metricsEnabled": true,
+            "snmpTrapsEnabled": false,
+            "snmpTrapsTarget": "192.168.24.254",
```

Comment:
Is there any meaning/use to this address? Or just a random example?

Reply:
This is just a random example, we can not leave this empty. The other obvious target would be 127.0.0.1.

Reply:
Perhaps these are most appropriate, then?  `The blocks 192.0.2.0/24 (TEST-NET-1), 198.51.100.0/24 (TEST-NET-2), and 203.0.113.0/24 (TEST-NET-3) are provided for use in documentation.` (https://tools.ietf.org/html/rfc5737). I have to admit that these blocks are not immediately recognized as dummy addresses by most people.

Mostly I just wanted to make sure it wasn't actually magic for our test automation or something; so :+1: